Formula/daemontools.rb

```diff
@@ -22,6 +22,40 @@ def install
     end
   end
 
+  def caveats; <<~EOS
+    You must create /service directory as root before starting svscan.
+    Then you can start svscan with following.
+    `sudo brew services start daemontools`
```

Comment:
Ident this by two spaces instead of using a backtick.

Reply:
Thanks for the review! I fixed at 69de5fd.